DataFormats/L1TGlobal/interface/GlobalObject.h

```diff
@@ -59,7 +59,8 @@ namespace l1t {
     gtExternal = 28,
     gtZDCP = 29,
     gtZDCM = 30,
-    ObjNull = 31,
+    gtHTMHF = 31,
+    ObjNull = 32,
```

Comment:
To the best of my understanding, and as written at the top of this file, existing entries should not be modified, and new entries should be assigned new numbers.
```suggestion
    ObjNull = 31,
    gtHTMHF = 32,
```

Reply:
Hi @missirol thank you! I misunderstood the description then, I thought taht ObjNull was not part of the problem and we simply needed to reassign a certain label to a different GT object. I don't fully understand when we should expect to have ObjNull (and why it should be a problem to assign that label to a new object, if the old one was never used...).
I can change this but I am wondering if it's not confusing to have ObjNull not as last one.